core/css/icons.scss

```
@@ -388,6 +388,11 @@ audio, canvas, embed, iframe, img, input, object, video {
 	@include icon-color('text', 'filetypes', #969696, 1, true);
 }
 
+.icon-file-white,
+.icon-filetype-text-white {
+	@include icon-color('text', 'filetypes', $color-white, 1, true);
+}
+
```

`-white` is not ideal as the button will have a primary color adapted to the theming, so whenever there is a bright theme used the white fixed does not fit anymore. I'd suggest something like this:

```suggestion
.icon-file-primary,
.icon-filetype-primary {
	@include icon-color('text', 'filetypes', $color-primary-text, 1, true);
}

```